src/main/java/in/projecteka/consentmanager/user/SignUpService.java

```diff
@@ -22,21 +23,14 @@
 import java.util.concurrent.ExecutionException;
 import java.util.function.Function;
 
+@AllArgsConstructor
 public class SignUpService {
 
     private final static Logger logger = Logger.getLogger(SignUpService.class);
+    public static final long JWT_TOKEN_VALIDITY = 5 * 60 * 60;
     private JWTProperties jwtProperties;
     private LoadingCache<String, Optional<String>> unverifiedSessions;
```

Comment:
Why not use Spring cache Abstraction instead of using Guava cache directly?

Reply:
Guava cache is also in memory cache, right? and does not work in a cluster.

Reply:
Apparently Spring inbuilt cache does not support value expiry by default, but guava cache does that, that's why Abhisek used that.

Yeah, we could use the abstraction, but not a mandatory now though.

Reply:
ok. I will put that in our tech debt list.